master/internal/api_experiment.go

```diff
@@ -1997,6 +2015,12 @@ func (a *apiServer) SearchExperiments(
 		ColumnExpr("extract(EPOCH FROM sum(coalesce(a.end_time, now()) - a.start_time))").
 		Where("a.task_id = trials.task_id")
 
+	bestTrialID := db.Bun().NewSelect().
+		Table("trials").
+		ColumnExpr("DISTINCT ON (experiment_id) experiment_id").Column("id").
+		Group("experiment_id").Group("id").
+		Order("experiment_id").Order("searcher_metric_value")
```

Comment:
it seems that `trials.searcher_metric_value` doesn't take into account the `smaller_is_better` prop on the experiment config, so this might result in the wrong trial being picked as the best trial for those experiments, so I think we need to keep searcher info and join it here.